src/network_core/peers.h

```diff
@@ -491,6 +498,11 @@ static bool peerConnectionNewlyEstablished(unsigned int i)
         // new connection has been established
         if (peers[i].isConnectedAccepted)
         {
+            if (peers[i].isIncommingConnection)
+            {
+                numberOfAcceptedIncommingConnection++;
+                ASSERT(numberOfAcceptedIncommingConnection < NUMBER_OF_INCOMING_CONNECTIONS);

```

Comment:
I think the operator should be "<=" instead of "<" here. `numberOfAcceptedIncommingConnection == NUMBER_OF_INCOMING_CONNECTIONS` is okay as well, right?

Reply:
agree. I will change it.